components/expenses/ExpandableExpensePolicies.js

```diff
@@ -15,12 +15,12 @@ const msg = defineMessages({
   },
 });
 
-const ExpandableExpensePolicies = ({ host, collective, ...props }) => {
+const ExpandableExpensePolicies = ({ host, organization, ...props }) => {
```

Comment:
I don't think we want to rename `collective` to `organization` here. For legacy reasons, in the vast majority of the codebase, "collective" is used as a generic word for all entities living in the `Collectives` table - regardless of their types. In new developments, we tend to use the word `Account` (which is the default in our public API).